deepchem/feat/molecule_featurizers/roberta_tokenizer.py

```diff
@@ -0,0 +1,74 @@
+from deepchem.feat import MolecularFeaturizer
+from deepchem.utils.typing import RDKitMol
+from typing import Dict, List
+try:
+  from transformers import RobertaTokenizerFast
+except ModuleNotFoundError:
+  raise ImportError('This class requires transformers.')
+  pass
+
+
+class RobertaFeaturizer(RobertaTokenizerFast, MolecularFeaturizer):
```

Comment:
Should we rename the file to roberta_featurizer.py to match the class name?

Reply:
@rbharath thoughts on naming convention? I think if we're going with featurizer a name change might be nice but we also want it to be clear that this Is a NLP-style tokenizer

Reply:
I think the name is probably OK as is for the file (`roberta_tokenizer.py` that is) for the reasons that Seyone mentions